vllm/executor/openvino_executor.py

```diff
@@ -270,14 +287,16 @@ def model_profile_run():
 
                 max_num_batched_tokens = self.scheduler_config.max_num_batched_tokens
                 max_num_seqs = self.scheduler_config.max_num_seqs
-                profiling_num_gpu_blocks = (max_num_batched_tokens + self.cache_config.block_size - 1) // self.cache_config.block_size
-                self.cache_config.num_gpu_blocks = profiling_num_gpu_blocks
+                cache_config = CacheConfig(block_size, device_memory_utilization, cpu_swap_space, cache_dtype)
+                cache_config.num_gpu_blocks = 1
```

Comment:
The idea was to evaluate memory pressure with minimal impact of KV cache, but do not use highest possible cache blocks number in the same time. Because in some cases usage of `max_num_batched_tokens` here may cause consuming more memory than GPU really has, and lead to hangs/OOO errors
I think we may revise this logic later and do something like `cache_config.num_gpu_blocks = min(free_gpu_memory_after_model_load, profiling_num_gpu_blocks*blocks*kv_cache_num)`